src/deck.py

```diff
@@ -1,7 +1,20 @@
 from card import Card
-
+import random
+import uuid
 
 class Deck:
+    suits = ['Hearts', 'Diamonds', 'Clubs', 'Spades']
```

Comment:
Mentioned this in the previous PR, but this breaks the existing UI since it assumes the suits and ranks are single characters. If the UI is broken, you'll need to fix it before merging this.